extensions/wikia/AdEngine/js/AdLogicPageViewCounter.js

```diff
@@ -1,4 +1,5 @@
 /*global define*/
+// TODO remove entire file: https://wikia-inc.atlassian.net/browse/ADEN-5728
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)